src/accidental.ts

```diff
@@ -545,94 +522,53 @@ export class Accidental extends Modifier {
   }
 
   protected reset(): void {
-    const fontScale = this.renderOptions.fontScale;
-    this.glyph = new Glyph(this.accidental.code, fontScale);
-    this.glyph.setOriginX(1.0);
-
-    if (this.cautionary) {
-      this.parenLeft = new Glyph(Tables.accidentalCodesOld('{').code, fontScale);
-      this.parenRight = new Glyph(Tables.accidentalCodesOld('}').code, fontScale);
-      this.parenLeft.setOriginX(1.0);
-      this.parenRight.setOriginX(1.0);
-    }
-  }
+    this.text = '';
 
-  /** Get width in pixels. */
-  getWidth(): number {
-    if (this.cautionary) {
-      const parenLeft = defined(this.parenLeft);
-      const parenRight = defined(this.parenRight);
-      const parenWidth =
-        parenLeft.getMetrics().width +
-        parenRight.getMetrics().width +
-        this.renderOptions.parenLeftPadding +
-        this.renderOptions.parenRightPadding;
-      return this.glyph.getMetrics().width + parenWidth;
+    if (!this.cautionary) {
+      this.text += this.accidental;
+      this.textFont.size = Tables.lookupMetric('Accidental.fontSize');
     } else {
-      return this.glyph.getMetrics().width;
+      this.text += Tables.accidentalCodes('{');
+      this.text += this.accidental;
+      this.text += Tables.accidentalCodes('}');
```

Comment:
Why is the shorthand code for left / right parentheses a left / right **brace**? Shouldn't they be `(` and `)` instead?

It looks like this has been around since v3.0.9: https://github.com/0xfe/vexflow/blob/00ec15c67ff333ea49f4d3defbd9e22374c03684/src/accidental.js#L452-L458

@0xfe should the accidental shorthand code for a left/right parenthesis be an actual left/right parenthesis? I suppose we can keep the braces around for backwards compatibility. Just curious about the original reasoning behind this. Or was it because using parentheses makes the code slightly harder to read, since function calls also use parentheses? :-)